gfx/GFX_Text.h

```diff
@@ -25,7 +25,7 @@ namespace gfx
 
 		TTF_Font* getFont() const { return textFont_; }
 
-		std::string const getString() { return textString_; }
```

Comment:
Return by reference to avoid copy of the string (also method can be const).
**MUST**

Reply:
done